docs/release-notes/remove-pt-callbacks.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Breaking Changes**
```

Comment:
Has this been coordinated with a particular release?

I know we're planning on a minor version bump in the next release, it might make sense to pull this breaking change out of this feature branch and land it separately (unless you know for a fact this feature branch will land in time).

Reply:
wait... this already landed... what's going on with this diff?

Reply:
wait... this already landed... what's going on with this diff?